manifests/06-operator.yaml

```diff
@@ -7,11 +7,11 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      name: cluster-samples-operator
+      app: samples-operator
   template:
     metadata:
       labels:
-        name: cluster-samples-operator
+        app: samples-operator
```

Comment:
i don't *think* this change has migration/upgrade implications (the deployment should just get stomped) but i'm not certain.....this might leave the old pods around when you change the label selector?  have you tested this?  Same thing for downgrade implications.

Reply:
to be honest this was me playing around with getting the prometheus instance in the openshift-monitoring namespace to pick up the metrics here (which I have still not gotten to work yet, though have another round of things to try before officially soliciting help) ... I was mimicking the OCM-O here ... if I can get it to work with `name: cluster-samples-operator` I'll switch back

Reply:
As a side note: I remember I also had problems making Prometheus to read metrics, make sure you can see the endpoint on Prometheus interface(there is a link called "Status" on the top of the interface). Other than that, be patient, metrics take time to show up there(5 minutes at least).

Reply:
I've switched it back and along with reordering some of the manifest files have made some progress.

And with @ricardomaraschini 's tip, I am at least seeing samples operator show up in the status/targets page, but the endpoint list is empty and shows 0/0 up.

I'll post more details in a new, non code review comment.

Reply:
I also had this problem(0/0). When I asked for help on monitoring channel they told me that it was something with the selector(it is like the ServiceMonitor not being able to select the Service from where to read the metrics). 

The only difference between our implementations on this regard is that you are using `matchLabels` while I am not. I don't think this could be causing the issue.

https://github.com/ricardomaraschini/cluster-image-registry-operator/blob/metrics/manifests/0000_90_cluster-image-registry-operator_02_operator-servicemonitor.yaml#L16

Reply:
I've removed the `matchLabels` piece and @ricardomaraschini and I are actively debugging